packages/server/graphql/public/mutations/createPDF.ts

```diff
@@ -0,0 +1,48 @@
+import puppeteer from 'puppeteer'
+import {PALETTE} from '../../../../client/styles/paletteV3'
+import {FONT_FAMILY} from '../../../../client/styles/typographyV2'
+import {MutationResolvers} from '../resolverTypes'
+
+const createPDF: MutationResolvers['createPDF'] = async (_source, {htmlContent}) => {
+  const browser = await puppeteer.launch({headless: 'new'})
+  const page = await browser.newPage()
+
+  await page.setContent(htmlContent, {
+    waitUntil: 'networkidle0' // wait until there are no more network connections so the images are loaded
+  })
+
+  await page.evaluate(() => {
+    // the buttons looks bad in the pdf and aren't needed, so remove them
+    const buttonsTable = document.querySelector(`#hideButtonsInPDF`)
```

Comment:
The buttons looks bad in the PDF so we need to remove them or update their styles: 

<img width="765" alt="Screenshot 2023-09-12 at 15 58 04" src="https://github.com/ParabolInc/parabol/assets/39854876/3dc00d36-3a20-4c8c-8ab7-054a59f6d8a2">

They're only useful in the web app, so I think we should remove them.

I don't like finding the elements by using their IDs, but I'm not sure of a better way.